app/components/Application/index.jsx

```diff
@@ -22,11 +22,11 @@ var Application = React.createClass({
     },
     render() {
         return <div>
-            <TopNav onMenuIconButtonTouchTap={this._onMenuIconButtonTouch} title={this.state.title}/>
             <SideNav ref='sideNav' />
             <div className='mainContent'>
                 <RouteHandler setTitle={this._setTitle}/>
             </div>
+            <TopNav onMenuIconButtonTouchTap={this._onMenuIconButtonTouch} title={this.state.title}/>
```

Comment:
Drop-down menu in `TopNav` can be seen but **not clickable** in Chrome.
But it works fine whatever the order is in Firefox.